indexing-service/src/main/java/io/druid/indexing/worker/WorkerTaskMonitor.java

```diff
@@ -60,15 +66,16 @@
   private final WorkerCuratorCoordinator workerCuratorCoordinator;
   private final TaskRunner taskRunner;
   private final ExecutorService exec;
-  private final List<Task> running = new CopyOnWriteArrayList<Task>();
+
```

Comment:
"The monitor implements {@link io.druid.query.QuerySegmentWalker}" part in Javadoc is no longer true.